src/Apps/Order/Routes/Shipping2/__tests__/Shipping2.jest.tsx

```diff
@@ -266,7 +266,7 @@ const getAllPendingOperationNames = (env: MockEnvironment) => {
 let mockTrackEvent: jest.Mock
 
 // FIXME: CI Timeouts likely due to fillAddressForm() duration...
-describe.skip("Shipping", () => {
+describe("Shipping", () => {
```

Comment:
n.b. this will be removed (skipped once again) after i have verified existing tests are passing.